core/src/main/kotlin/org/jetbrains/kotlinx/dataframe/impl/codeGen/CodeGeneratorImpl.kt

```diff
@@ -483,7 +476,7 @@ internal class CodeGeneratorImpl(typeRendering: TypeRenderingStrategy = FullyQua
                 append("\n}")
             }
         } else {
-            ""
+            " { }"
```

Comment:
is here an empy symbol before brackets?

Reply:
yes, it's intentional so that generated code is `interface Name { }`